.circleci/config.yml

```diff
@@ -400,12 +391,12 @@ workflows:
       - test
       - assemble-sample-app
 
-  integration-test:
+  deploy:
     when:
       not:
         equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
     jobs:
-      - integration-tests-build: *release-tags-and-branches
+      - integration-tests-build: *release-branches
```

Comment:
`release-tags-and-branches` was running this twice, on the PR and on the tag created by `revenuecat/tag-current-branch`, but both results should be the same, so I just changed it to run on the branch and removed the filter